os/cluster-architectures.md

```diff
@@ -60,22 +49,43 @@ coreos:
         ExecStart=/usr/bin/systemctl enable docker-tcp.socket
 ```
 
+This file is used to provision your local CoreOS machine on it's first boot. It enables the docker sets up and enables the docker socket in systemd.
+
+Using the Container Linux Config Transpiler, or `ct`, ([download][ct-getting-started]) convert the above yaml into an [ignition][ignition-getting-started]. Once you have the ignition configuration file, pass it to your provider ([complete list of supported ignition platforms][ignition-supported]).
+
+Once the local VM is running, tell your Docker binary to use the remote port by exporting an environment variable and start running Docker commands:
+
+```
+$ export DOCKER_HOST=tcp://localhost:2375
+$ docker ps
+```
+
+This avoids potentially breaking differences between your local development environment and where the container will *actually* run in production.
```

Comment:
breaking differences is a weird phrase IMO. is there another way to get this across?